lib/pronto/config.rb

```diff
@@ -4,7 +4,7 @@ def initialize(config_hash = ConfigFile.new.to_h)
       @config_hash = config_hash
     end
 
-    %w(github gitlab).each do |service|
+    %w(github gitlab text_formatter).each do |service|
```

Comment:
`text_formatter` isn't a service, maybe that name should be updated to something like `nested_config`?